### DISCUSSION TITLE
No firmware in the TGZ file?

### DISCUSSION THREAD

**Discussion by jopuisenki** [alienatedsec/yi-hack-v5#288] [General]
Hello,

when i unzip yi_dome_1080p_0.4.0.tgz, there is no firmware type : home_y19 & rootfs_y19? is it normal?
i'm currently in V4 but reall intersted by RTSP function.

Regards,

Enki

**Comment by jopuisenki** [comment]
Ok i just reallized you need to upgrage in 0.3.8 before to start,
I hpe my mistake could help the others!

**Reply by pvcraven** [reply]
Thanks for posting. It would be GREAT if the official how-to docs included this step.
